Library/Formula/denominator.rb

```diff
@@ -1,11 +1,12 @@
 class Denominator < Formula
   # This is a shell script wrapping a java jar; download as single file using nounzip
```

Comment:
This comment can probably be removed now that the URL has the `.jar` extension in it.